PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/AzurePlugin.java

```diff
@@ -270,27 +185,16 @@ public void run() {
                 });
     }
 
-    private void telemetryAI() {
-        ModuleManager.getInstance(myProject).getModules();
-    }
-
     public String getComponentName() {
         return "MSOpenTechTools.AzurePlugin";
     }
 
-    // currently we didn't have a better way to know if it is in debug model.
-    // the code suppose we are under debug model if the plugin root path contains 'sandbox' for Gradle default debug path
-    private boolean isDebugModel() {
-        return PluginUtil.getPluginRootDirectory().contains("sandbox");
```

Comment:
Does it also depend on Java? Does Rider have debug mode?

Reply:
Ok, will return